appendices/ocr-tesseract-addon.md

```diff
@@ -0,0 +1,19 @@
+# OCR Tesseract Addon
+
+The OCR Tesseract Addon is not a part of a G1ANT.Studio platform download and you need to install it manually. Follow these simple steps:
```

Comment:
shouldn't it be "the G1ANT.Studio platform"?

Reply:
I'm not sure, I think it's a "(...) download"

Reply:
ok, you decide :) I approved it